public_html/wp-content/mu-plugins/blocks/assets/src/shared/versatile-select/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import Select from 'react-select';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+const { BaseControl, Button } = wp.components;
+const { Component } = wp.element;
+const { __ } = wp.i18n;
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+class VersatileSelect extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			selectedOptions: null,
+		};
+
+		this.render = this.render.bind( this );
```

Comment:
Why bind render here?

Reply:
I guess I don't fully understand when these binding statements are necessary and when they're not. I probably got this from some component in the Gutenberg codebase.